packages/dev-frontend/src/components/Farm/views/Yield.tsx

```diff
@@ -28,6 +28,13 @@ export const Yield: React.FC = () => {
   const hasZeroValue = remainingLiquidityMiningLQTYReward.isZero || totalStakedUniTokens.isZero;
   const lqtyTokenAddress = addresses["lqtyToken"];
   const uniTokenAddress = addresses["uniToken"];
+  const [daysRemaining, setDaysRemaining] = useState<Decimal | null>(null);
+
+  useEffect(() => {
+    const secondsRemaining = remainingLiquidityMiningLQTYReward.div(liquidityMiningLQTYRewardRate);
+    const days = secondsRemaining.div(60 * 60 * 24);
+    setDaysRemaining(days);
+  }, [liquidityMiningLQTYRewardRate, remainingLiquidityMiningLQTYReward]);
```

Comment:
I think now that it no longer uses `Date.now()`, the useState & useEffect are no longer needed. It could just be:
```typescript
const secondsRemaining = remainingLiquidityMiningLQTYReward.div(liquidityMiningLQTYRewardRate);
const daysRemaining = secondsRemaining.div(60 * 60 * 24);
```

Reply:
Ah the Liquity selector forces a component update if any of the values change right?